test/stdlib/StringDescribing.swift

```diff
@@ -22,4 +22,19 @@ StringDescribingTestSuite.test("String(reflecting:) should include extra stuff i
   expectEqual(privateName.suffix(5), ").Foo")
 }
 
+StringDescribingTestSuite.test("String(describing:) shouldn't include local discriminator") {
+  class LocalType {}
+  let t = LocalType()
+  expectEqual(String(describing: type(of: t)), "LocalType")
+}
+
+StringDescribingTestSuite.test("String(reflecting:) shouldn't include local discriminator") {
```

Comment:
…`String(reflecting:)` should give a unique answer for every unique type.

Reply:
Should assertion message be reworded?

Reply:
Yes, like the one for private discriminators.